packages/client/src/StreamrClientError.ts

```diff
@@ -0,0 +1,7 @@
+export class StreamrClientError extends Error {
+    constructor(message: string, readonly errorCode?: 'NO_STORAGE_NODES') {
```

Comment:
Are codes a good way to identify errors? Or is `instanceof` preferrable? Research.

Reply:
Fixed in fafafd2a5e2d44ddf16879b02f512d460a4be828